### DISCUSSION TITLE
Make a lite version of Directory Lister

### DISCUSSION THREAD

**Discussion by ilikemacsalot** [DirectoryLister/DirectoryLister#962] [Ideas & Features]
Make a smaller version of Directory Lister.

**Comment by PHLAK** [comment]
Thanks for the suggestion. What problems are you running into that makes you want a "lite" version?

I already think Directory Lister is pretty lite as-is since that was kind of the point from the beginning (e.g. no database, no authentication).

**Comment by avocadowebservices** [comment]
As said above, I do agree. Because if you unzip and it comes to 1,325 files. By lite, it would mean like few files. For example:

https://github.com/halgatewood/file-directory-list

Only 2 files, that is called lite-version 

Just sayin'
